modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/utilities/utilities.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {fetch} from 'frontend-js-web';
+
+export const HEADERS = new Headers({
+	Accept: 'application/json',
+	'Content-Type': 'application/json',
+});
+
+const SKUS_ENDPOINT = '/o/headless-commerce-admin-catalog/v1.0/skus';
+const DIAGRAMS_ENDPOINT = '/o/headless-commerce-admin-catalog/v1.0/products';
```

Comment:
These are going to be flakey since we are depending on the `v1.0`. Can we get these endpoints from the backend or have a way to ensure that these endpoints won't break if headless version is at `v1.1`

Reply:
I'd rather manage this case with some logic here, because we will need to support all versions if there will be multiple versions in the future

Reply:
Yeah we need to make sure these endpoints are dynamic and not hard coded for a specific version of the catalog endpoint. Committing these with an explicit v1.0 means that if the catalog module is bumped to 1.1(which sometimes modules get bumped without us realizing it), it will cause bugs and likely break your product.